lib/rubocop/ast/processed_source.rb

```diff
@@ -27,9 +27,10 @@ def self.from_file(path, ruby_version, parser_engine: :parser_whitequark)
       end
 
       def initialize(source, ruby_version, path = nil, parser_engine: :parser_whitequark)
+        parser_engine = parser_engine.to_sym if parser_engine.is_a?(String)
```

Comment:
I recall `.to_sym` is defined on symbols, so you don't really need the `if` here.